### PR TITLE
CD-360332 github actions

### PR DESCRIPTION
**Pull request by ysedliarou**
https://coupadev.atlassian.net/browse/CD-360332

Github actions
Unit tests fix

### CODE REVIEW

File: test/worker_test.rb [hunk 1]
```diff
@@ -227,17 +229,20 @@ def self.perform
...
+    job.worker = @worker
     @worker.perform(job)
     assert_equal 1, Resque::Failure.count
     assert_equal 1, SimpleFailingJob.exception_count
   end
 
   it "can peek at failed jobs" do
-    10.times { Resque::Job.create(:jobs, BadJob) }
+    n = 10
+    n.times do |i|
```
Comment (abdulchaudhrycoupa) [inline test/worker_test.rb:240]: @ysedliarou minor: you can just say `10.times` instead of using a local variable

File: test/worker_test.rb [hunk 2]
```diff
@@ -880,6 +888,7 @@ def self.perform
   end
 
   it "remove failed queue" do
+    skip ".remove_queue has no implementation"
```
Comment (abdulchaudhrycoupa) [inline test/worker_test.rb:891]: @ysedliarou lets create a ticket and add the ticket number here

File: test/worker_test.rb (hunk 1 above)
Comment (ysedliarou) [reply test/worker_test.rb:240]: it's used lateter on for assertions

**Review by abdulchaudhrycoupa — APPROVED** [review]

**Comment by ysedliarou** [comment]
[lockit v5.0.2] merged via Lockit CLI on 2023-06-07 10:10:22.971548 -0700 PDT m=+8.514743163

**Pull request merged**